doc/Client-Portal Protocol.md

```diff
@@ -506,7 +507,11 @@ This a format for client reconnects.
          parameters: object(VideoParametersSpecification)/*If any video parameters are wanted to be re-specified*/
                      | undefined/*If the video parameters of the ongoing subscription are wanted to be kept*/
         }
-**ResponseData**: undefined if **ResponseStatus** is “ok”.
+**ResponseData**: undefined or object(BWEResult) for "querybwe" if **ResponseStatus** is “ok”.
+  object(BWEResult)::
+    {
+      bitrate: number(bps) /*Send side estimated bitrate if enabled*/
```

Comment:
Did you consider to rename it as `estimatedBitrate`? From JavaScript code or signaling data perspective, object doesn't have the name `BWEResult`, it's just an object with a property `bitrate`. Not sure if more bitrate info will be required in the future.